maven-embedder/src/main/java/org/apache/maven/cli/event/ExecutionEventLogger.java

```diff
@@ -133,6 +135,20 @@ public void sessionEnded( ExecutionEvent event )
                 logReactorSummary( event.getSession() );
             }
 
+            ILoggerFactory iLoggerFactory = LoggerFactory.getILoggerFactory();
+
+            if ( iLoggerFactory instanceof MavenSlf4jWrapperFactory )
```

Comment:
I'm missing an `else` here. Is there a situation where this condition is _not_ going to be `true`? If there is, the `fl` switch doesn't work? If so, maybe we should at least say that it's not going to work.